jerry-debugger/jerry-client.py

```diff
@@ -0,0 +1,22 @@
+import socket
+import sys
+
+port = 5001 
```

Comment:
1. there is a trailing whitespace at the line of this line. Use git diff before commit and/or git show before push. Trailing whitespaces are marked with a "red rectangle".
2. Use capitalized constants. For example, PORT = 5001